src/tests/teamcity-event-listener.tests.csproj

```diff
@@ -28,6 +29,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <LangVersion>3</LangVersion>
```

Comment:
My understanding is that the LangVersion 3 setting indicates that what we normally call C# 5.0 is being used. If correct, it's a new restriction on what compilers can be used by contributors. Is it necessary?